cmd/delete/matchable_execution_queue_attribute.go

```diff
@@ -0,0 +1,81 @@
+package delete
+
+import (
+	"context"
+
+	"github.com/flyteorg/flytectl/cmd/config"
+	sconfig "github.com/flyteorg/flytectl/cmd/config/subcommand"
+	"github.com/flyteorg/flytectl/cmd/config/subcommand/executionqueueattribute"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+const (
+	executionQueueAttributesShort = "Deletes matchable resources of execution queue attributes"
+	executionQueueAttributesLong  = `
+Deletes execution queue attributes for given project and domain combination or additionally with workflow name.
+
+Deletes execution queue attribute for project and domain
+Here the command delete execution queue attributes for project flytectldemo and development domain.
+::
+
+ flytectl delete execution-queue-attribute -p flytectldemo -d development 
+
+
+Deleting execution queue attribute using config file which was used for creating it.
+Here the command deletes execution queue attributes from the config file cra.yaml
+eg:  content of cra.yaml which will use the project domain and workflow name for deleting the resource
+
+::
+
+ flytectl delete execution-queue-attribute --attrFile era.yaml
+
+
+.. code-block:: yaml
+	
+	domain: development
+	project: flytectldemo
+	tags:
+		- foo
+		- bar
+		- buzz
+		- lightyear
+
+Deleting execution queue attribute for a workflow
+Here the command execution queue attributes for a workflow
```

Comment:
this looks like it's missing a second half?

Reply:
Done.